lib/cli.ex

```diff
@@ -30,6 +33,7 @@ defmodule Onigumo.CLI do
     COMPONENT\tOnigumo component to run, available: #{components}
 
     OPTIONS:
+    -h, --help\t\tprint this help
```

Comment:
Capitalized, this sentence would be more consistent.

```suggestion
    -h, --help\t\tPrint this help
```